desk/app/reel.hoon

```diff
@@ -94,15 +159,26 @@
       =/  new  (rap 3 (scot %p our.bowl) '/' token ~)
       :-  (~(put by md) new metadata)
       (~(put by id) new new)
-    `this(our-metadata new-md)
-      %3
-    `this(state [%4 vic.old civ.old our-metadata.old outstanding-pokes.old ~ ~])
-      %2
-    `this(state [%4 vic.old civ.old our-metadata.old ~ ~ ~])
-      %1
-    `this(state [%4 'https://tlon.network/lure/' ~loshut-lonreg ~ ~ ~ ~])
-      %0
-    `this(state [%4 'https://tlon.network/lure/' ~loshut-lonreg ~ ~ ~ ~])
+    :*  %5
+        vic.old
+        civ.old
+        *contact:t  ::  profile
+        new-md
+        open-link-requests.old
+        open-describes.old
+        stable-id
+        ~  ::  subs
+    ==
+  ?>  ?=(%5 -.old)
+  =.  state  old
+  :_  this
+  %-  murn  :_  same
+  ^-  (list (unit card))
+  :~  ?:  (~(has by wex.bowl) /groups [our.bowl %groups])  ~
+      `[%pass /groups %agent [our.bowl %groups] %watch groups-path]
+    ::
+      ?:  (~(has by wex.bowl) /contacts [our.bowl %contacts])  ~
+      `[%pass /contacts %agent [our.bowl %contacts] %watch contacts-path]
```

Comment:
If we're gonna run these unconditionally, might be nicer inside an `+inflate-io` to make that more explicit.

Reply:
And, do we not want to call `subscribe:s` here too? We do so for `%kick` handling. I assume we want the subscriptions to end up in library state consistently.